src/main.cpp

```diff
@@ -86,7 +86,7 @@ static void NonCtrlHandler(int a_Signal)
 		case SIGTERM:
 		{
 			// Server is shutting down, wait for it...
-			cRoot::Get()->Stop();
```

Comment:
The signal handlers are set on line 297. Simply move that to after `cRoot` has been constructed and unset it before destroying `cRoot`.

Having a weird half singleton, half static variable object doesn't make sense to me.

Reply:
I don't want to mutate the signals stuff after startup since it feels like artificially creating a problem for ourselves and then having to write more code to solve it. If you'd prefer I can try enclosing the statics into their own support class away from cRoot?